### PR TITLE
Added: Logger option in MySQLConnection

### PR DESCRIPTION
**Pull request by stillalivx**
Hey, @eveningkid! I was wondering if is possible to add this option to MySQL connection. With it, users could control the log messages that MySQL driver shows at the console. If you want more information, denodrivers has [this](https://github.com/denodrivers/mysql#logging) little section. 
I use it when I want to run some tests and the log data makes a caos with the test's messages

This is how it works:
```typescript
const connector = new MySQLConnector({
  ... // essential data connection
  logger: { // Logger options (it's not only enable)
    enable: false
  }
});
```

### CODE REVIEW

File: deps.ts [hunk 1]
```diff
@@ -8,8 +8,11 @@ export { camelCase, snakeCase } from "https://deno.land/x/case@v2.1.0/mod.ts";
 
 export {
   Client as MySQLClient,
+  configLogger as MySQLConfigLogger,
   Connection as MySQLConnection,
+  log as MySQLLogs,
```
Comment (eveningkid) [inline deps.ts:13]: This one is exported but never used

File: deps.ts [hunk 2]
```diff
@@ -8,8 +8,11 @@ export { camelCase, snakeCase } from "https://deno.land/x/case@v2.1.0/mod.ts";
 
 export {
   Client as MySQLClient,
+  configLogger as MySQLConfigLogger,
```
Comment (eveningkid) [inline deps.ts:11]: I think it's misleading to use PascalCase here, it makes me think it's a class and not a function.

Maybe name it `configMySQLLogger` instead?

File: lib/connectors/mysql-connector.ts [hunk 3]
```diff
@@ -1,4 +1,9 @@
-import { MySQLClient, MySQLConnection } from "../../deps.ts";
+import {
+  LoggerConfig,
```
Comment (eveningkid) [inline lib/connectors/mysql-connector.ts:2]: Shouldn't this one be `import type { LoggerConfig } ...`? Not sure

**Review by eveningkid — CHANGES_REQUESTED** [review]
No problem, totally legit :)

Just have a look at the comments and we can merge.

Thank you!

**Comment by stillalivx** [comment]
Okay, I think it's ready.

**Comment by eveningkid** [comment]
All good, let's merge :)

Thank you!

**Review by eveningkid — APPROVED** [review]

**Pull request merged**